src/python/k4a/docs/building.md

```diff
@@ -79,14 +79,14 @@ The following tools are optional:
    folder named libk4a\<ver\>, where \<ver\> is the version major.minor of the k4a library
    (e.g. _libs constains libk4a1.4/libdepthengine.so.2.0).
    
-    >**Note:** To create a cross-platform wheel, the .dll for Windows and the .so for Linux need to
-    >copied into the folder. Python will look for "k4a.dll" in Windows and "libk4a.so" in Linux.
+    >**Note:** To create a Linux-only wheel, the the .so for Linux need to copied into the folder. 
+    >Python will look for "libk4a.so" in Linux which will also load the depth engine dll.
 
     >**Note:** An example of the files in _libs for Linux are: 
     >- libk4a.so, a soft link to libk4a.1.4
```

Comment:
These are the files on Linux (looks like we are missing file suffixes in the instructions)
libk4a.so (link) -> libk4a.so.1.4
libk4a.so.1.4 (link) -> libk4a.so.1.4.0
libk4a.so.1.4.0

Reply:
I modified the wording slightly in the most recent commits. See if it's clearer.